CHANGELOG.md

```diff
@@ -2,6 +2,25 @@
 
 All notable changes to the DTS project will be documented in this file.
 
+## v1.2.20 - 2024-03-19
```

Comment:
@mkopec can you do sanity check of the changelog content?

Reply:
@TomaszAIR LGTM